dart/lib/src/protocol/sentry_event.dart

```diff
@@ -285,52 +280,52 @@ class SentryEvent {
     }
 
     if (level != null) {
-      json['level'] = level.name;
+      json['level'] = level!.name;
```

Comment:
Hm, level should probably be non-nullable with a default value

Reply:
level is inferred on the server if not available, we don't need to set it then.